products/blender.md

```diff
@@ -11,6 +11,7 @@ changelogTemplate: https://www.blender.org/download/releases/{{"__RELEASE_CYCLE_
 releaseDateColumn: true
 activeSupportColumn: true
 eolColumn: Critical bug fixes
+versionCommand: blender --version
```

Comment:
Must be moved before the `releasePolicyLink` as per [our style guide](https://github.com/endoflife-date/endoflife.date/wiki/Style-Guide).